spec/features/newflow/student_signup_flow_spec.rb

```diff
@@ -104,30 +105,32 @@ module Newflow
     end
 
   example 'arriving from Tutor (a Doorkeeper app)' do
```

Comment:
Are we leaving Tutor here as an inside joke 😄

Reply:
I didn't want to get too much into cleanup... I don't even know if the Message API and records are/were ever used.

Reply:
Been there, done that. I agree, let's just get off EOL stuff for now.